.ci/trial-json-schema.json

```diff
@@ -45,7 +39,7 @@
       "$ref" : "#/definitions/SpringCloud"
     }
   },
-  "required" : [ "name", "id", "version", "rootUser", "tenant_id", "ui_client_id", "spring_profile", "spring_config_label", "site_service", "role_service", "practitioner_service", "init_service", "spring_cloud" ],
```

Comment:
I will open a PR in mts-services to remove these values from the json schema tool